prov/efa/src/efa_strerror.c

```diff
@@ -101,6 +102,22 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
+	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
+		return "\nThis error can be caused by a crashed peer. "
+			   "Please verify the receiver status";
```

Comment:
At this point do we have any information as to which peer we are trying to address?  I think that would be very helpful to the user.

Reply:
That's the plan. We want to log the peer instance id (or some identifier) to help debug. That's the next step.

Reply:
Also for that I need to evaluate what the best place is - this function as-is is not capable to accept additional metadata from the caller, i.e. static/stateless.
We might do it at the place of erroring.